FAQ.md

```diff
@@ -106,15 +106,36 @@ Views are buffered, so don't panic if the view counter stays the same after you
 
-So you would need:
- * **CPU** 1 core if you don't enable transcoding, 2 at least if you enable it (works with 1 but this is really slow)
- * **RAM** 1GB
- * **Storage** Completely depends on how many videos your users will upload
+As a real life example, the PeerTube demonstration server [https://peertube.cpy.re](https://peertube.cpy.re) runs on 2 vCores and 2GB of RAM. Average consumption is:
+ * **CPU**: nginx ~ 20%, peertube ~ 10%,   postgres ~ 1%, redis ~ 3%
+ * **RAM**: nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
 
+### CPU
+Except for video transcoding, a PeerTube instance is not really cpu bound. Neither nginx, peertube, postgres nor redis require a lot of cpu. If it were only for those, once could easily get by with just one thread.
+
+You will hugely benefit from at least a second thread though, because of transcoding. Transcoding _is_ very cpu intensive. It serves two purposes on a PeerTube instance: it ensures all videos can be played optimally in the web interface, and it generates different resolutions for the same video. PeerTube support for offloading transcoding to other machines is planned, but not yet implemented. See https://github.com/Chocobozzz/PeerTube/issues/1192 .
+
+### RAM
+1 Gb of RAM should be plenty for a basic PeerTube instance. The only reason you'd want significantly more would be to RAM cache very popular video fragments.
+
+### Storage
+There are two important angles to storage: disk space usage and sustained read speed.
+
+To make a rough estimate of your disk space usage requirements, you want to know the answer to three questions:
+- What is the size of the video source archive?
+- Do you want to enable transcoding? If so, do you want to provide multiple resolutions per video? Try this out with a few videos and you'll get an idea of a multiplication factor.
+- Which sharing mechanisms do you want to enable? Just WebTorrent, or also HLS with p2p? If you want both, this will double your storage needs.
+
+In terms of read speed, you want to make sure that you can saturate your network uplink serving PeerTube videos. This should not be a problem with ssd disks. With traditional hard disks though it is really worth checking! Typical sustained read rates for a well tuned system with a 7200rpm hard disk should hover around 120 MB/s. This should about saturate a 1 Gbit/s network uplink.
+
+### Network
+A rough estimate of a traditional server's video streaming network capacity is usually quite straightforward. You simply divide your server's available bandwidth by the average bandwidth per stream, and you have an upper bound.
+
+Take a server for example with a 1 Gbps/s uplink for example pushing out 1080p60 streams at 5 Mbps per stream. That means the absolute theoretical upper capacity bound is 200 simultaneous viewers if your server's disk i/o can keep up. Expect a bit less in practice.
```

Comment:
`1 Gbps` or `1 Gbit/s` - I suggest the use of the `Gbps` form.

Reply:
Just updated the pull request to reflect your suggestion. Rereading, turns out I misread it and changed everything to consistently use Mbit/s and Gbit/s. That helps keep it consistent with how sustained read rates from disk are noted though...

Reply:
Thank you!